pxd_fastpath.c

```diff
@@ -1545,7 +1308,7 @@ void pxd_fastpath_adjust_limits(struct pxd_device *pxd_dev, struct request_queue
 		if (!bdev || IS_ERR(bdev)) {
 			printk(KERN_ERR"pxd device %llu: backing block device lookup for path %s failed %ld\n",
 				pxd_dev->dev_id, pxd_dev->fp.device_path[i], PTR_ERR(bdev));
-			continue;
+			goto out;
```

Comment:
If this fails on a block device for any reason, then the queue parameters between px volume and the backing device may not be compatible and may result in failed IO. Hence failing to native path.